lib/fake_async.dart

```diff
@@ -68,6 +68,10 @@ class FakeAsync {
   /// All timers created within [run].
   final _timers = <_FakeTimer>{};
 
+  /// Debugging information for all pending timers.
+  List<String> get pendingTimersDebugInfo =>
+      _timers.map((timer) => '${timer.debugInfo}').toList(growable: false);
```

Comment:
We don't need the extra `'${}'` here. `debugInfo` is already a String.

Reply:
Ya just copy/pasted from quiver, ill update though